exchange/exchange.go

```diff
@@ -79,6 +81,24 @@ type bidResponseWrapper struct {
 	bidder       openrtb_ext.BidderName
 }
 
+type UUIDGenerator interface {
```

Comment:
Nitpick: Please change the name to be specific to BidID, such as `BidIDGenerator`.

Reply:
Renamed